lib/semantics/resolve-labels.cc

```diff
@@ -209,6 +210,14 @@ const parser::CharBlock *GetStmtName(const parser::Statement<A> &stmt) {
   return nullptr;
 }
 
+template<typename A>
+static constexpr bool LabelDoTargetAcceptedInDirectSubscopeOn(
+    const parser::Statement<A> &statement) {
+  return (std::is_same_v<A, parser::EndIfStmt> ||
```

Comment:
Needless parentheses; could also be written by defining a `std::tuple<>` of these types and using `common::HasMember`.

Reply:
Thanks, that makes even more sens now that there are more `End..Stmt` to handle.